lib/segment/tests/integration/payload_index_test.rs

```diff
@@ -66,6 +66,7 @@ macro_rules! ensure {
 
 const DIM: usize = 5;
 const ATTEMPTS: usize = 20;
+const RNG_SEED: u64 = 42;
```

Comment:
had some failing tests there that I had trouble fixing due to the lack of determinism